cluster-autoscaler/processors/nodegroupset/compare_nodegroups.go

```diff
@@ -124,8 +126,10 @@ func IsCloudProviderNodeInfoSimilar(n1, n2 *schedulernodeinfo.NodeInfo, ignoredL
 		switch kind {
 		case apiv1.ResourceMemory:
 			// For memory capacity we allow a small tolerance
-			memoryDifference := math.Abs(float64(qtyList[0].Value()) - float64(qtyList[1].Value()))
-			if memoryDifference > MaxMemoryDifferenceInKiloBytes {
+			difference := absSub(qtyList[0], qtyList[1])
```

Comment:
Could just use  `math.Abs`?

Reply:
I wanted to keep all of the quantities as `resource.Quantity`'s, so this helper allows us to do that and reduce the conversions to/from integers to reduce the likelihood of a mistake being made there